src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
It's better to use `fr` units instead of `px` for grid-template-columns. It makes your layout more flexible. For example, you could write `grid-template-columns: repeat(1, 1fr);`